db/migration/cmd/main.go

```diff
@@ -27,14 +27,20 @@ func main() {
 	sslmode := viper.GetString("dbsslmode")
 
 	var forceVersion int
+	var direction string
 	var err error
-	if len(os.Args) > 1 {
-		forceVersion, err = strconv.Atoi(os.Args[1])
+	if len(os.Args) > 2 {
+		direction = os.Args[1]
```

Comment:
based on the command from the Dockerfile, shouldn't direction be the 2nd argument and forceVersion be the first?

Reply:
Totally forgot to switch the order in the Dockerfile as well as the makefile command, fixed!